workbench/vite.config.js

```diff
@@ -15,7 +15,7 @@ export default defineConfig({
   build: {
     mode: process.env.MODE,
     sourcemap: 'inline',
-    minify: process.env.MODE !== 'development',
+    minify: false,
```

Comment:
Is this related to the issues addressed in this PR?

Reply:
Yeah, good question. It is related because now that we're logging error traceback from browser exceptions, it's much nicer to have un-minified code in that traceback. Otherwise we have to figure out how to decode it with source maps.

Plus the size difference in the bundle is completely negligible in the context of all the invest & electron binaries. Minification is really just meant for app's distributed over the network, I think.